task/olot-oci-ta/0.1/olot-oci-ta.yaml

```diff
@@ -0,0 +1,339 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: oci-copy-oci-ta
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, konflux
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: oci-artifact
+spec:
+  description: Given a file in the user's source directory, copy content from
```

Comment:
Would be nice to have more detailed description.

Reply:
I expanded the task's `description`